### ISSUE TITLE
Support Site - SiteImprove Report Accessibility Issues

### ISSUE THREAD

**Issue by mheppler** [IQSS/dataverse.harvard.edu#57]
SiteImprove finished crawling the site and generated a report. Here are some issues that it found. Will update with progress. These issues were spun off from _Reference website for Harvard Dataverse #26_.

<img width="1092" alt="Screen Shot 2020-01-24 at 2 04 50 PM" src="https://user-images.githubusercontent.com/687227/73096751-8df60480-3eb3-11ea-952a-c9f2739e91ad.png">

- Color contrast is insufficient
- Link text used for multiple different destinations
- Redundant WAI-ARIA attribute
- Content not included in landmarks
- Link text used for multiple different destinations
- Non-distinguishable landmarks
- HTML is used to format content
- Heading is missing text
- iFrame is missing a title
- Headings are not nested properly
- "Bold" tag used to format text
- Link identified only by color
- Heading is missing text
- No top-level heading on the page
- Link text is too generic
- No top-level heading on the page

**Comment by mheppler** [comment]
Fixed all the "Link text used for multiple different destinations" that was reported on 13 pgs -- For Journals is the page with most instances at 6.

- "Contact Us" link in footer was outdated (included "aboutharvarddataverse")
- "Search the Harvard Dataverse Support site" link in footer was outdated
- "Social Science Sharing Network" and "Cousijn et al., 2017" links were in the For Journals content text twice, with different URL's

SiteImprove reports this fixed can increase our accessibility score by "0.32 points".

**Comment by mheppler** [comment]
Here are the remaining issues from the original list. These will require changes to the OpenScholar theme (CSS) or template code (LAYOUT).

**A + AA Errors and Warnings**

CSS
- Color contrast is insufficient AA (navbar and "section nav" sidemenu)
- Link identified only by color A

LAYOUT
- Content not included in landmarks A
- Non-distinguishable landmarks A (site navbar uses two nav tags, one for logo, and one for the primary menu)
- Heading is missing text A (homepage content widget do not have titles, but still display heading tags in the code that are empty)
- iFrame is missing a title A (homepage video, we didn't give the widget any title text, but adding it did not add a title attribute to the iframe tag)
- No top-level heading on the page A (homepage has no h1, because the tagline is wrapped in an h2 due to the OS theme not allowing you to add an h1 to the widget content, it is changed to a strong tag by OS)

**AAA Errors and Warnings**

CSS
- Color contrast is insufficient AAA (blue and orange links in content and "section nav" sidemenu)

LAYOUT
- Heading is missing text AAA (for whatever reason, the same headings are called out as both A and AAA errors)
- No top-level heading on the page AAA (for whatever reason, the same headings are called out as both A and AAA errors)

**Comment by TaniaSchlatter** [comment]
Here are colors that will pass standards:

Gray text to:
<img width="1253" alt="Screen Shot 2020-01-29 at 11 58 46 AM" src="https://user-images.githubusercontent.com/15386424/73379145-bf921580-428f-11ea-8b3c-a6a70afbf3a9.png">

Blue text to:
<img width="1298" alt="Screen Shot 2020-01-29 at 11 58 07 AM" src="https://user-images.githubusercontent.com/15386424/73379146-bf921580-428f-11ea-95fa-d1dda63019e5.png">

Orange text to:
<img width="1229" alt="Screen Shot 2020-01-29 at 11 56 59 AM" src="https://user-images.githubusercontent.com/15386424/73379147-bf921580-428f-11ea-99b8-3d59137412e5.png">

**Comment by mheppler** [comment]
FYI... fixed some broken links found in the Quality Assurance report...

- _Getting Started_... Bad format for "OpenScholar at Harvard" link, missing forward slash, fixed

- _Harvard Dataverse Policies_... Vague "here" link text and 404 broken link, so I removed the paragraph all together (seems more appropriate for Installation Guide if we want to add it back some where, with a working link, which I couldn't find)...

  > Other Dataverse installations are free to use these as templates for setting up their own policies, terms of use, and guidelines. If interested in using these documents as templates, you can fork them from our GitHub <a data-url="https://github.com/IQSS/dataverse.org/tree/master/docs/best_practices/source/harvard-policies" href="https://github.com/IQSS/dataverse.org/tree/master/docs/best_practices/source/harvard-policies">here</a>.

- _Privacy Policy_... changed link text and fixed broken URL to "Google Privacy & Terms" (https://policies.google.com/) rather than having two separate links for the policy and terms on the same Google site

**Comment by djbrooke** [comment]
Closing this. Many (all?) of these issues cannot be fixed unless we start editing the OS template, which we do not want to do. We will let OS address these, as either part of the Drupal 8 upgrade or through their other efforts.
